pkg/apiserver/api_installer.go

```diff
@@ -61,6 +61,19 @@ type documentable interface {
 	SwaggerDoc() map[string]string
 }
 
+// toDiscoveryKubeVerb maps an action.Verb to the logical kube verb, used for discovery
+var toDiscoveryKubeVerb = map[string]string{
+	"LIST": "list",
+	"POST": "update",
+	"DELETECOLLECTION": "delete",
```

Comment:
ptal at the 2nd commit. I guess that comment was wrong then as well. Copied the list from there.